src/index.js

```diff
@@ -1,8 +1,28 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
+import 'bootstrap/dist/css/bootstrap.min.css'; // Import Bootstrap CSS
```

Comment:
Where is this file? It's not committed in this PR so when I try to pull your branch and run it, I'm having errors